packages/table/src/spectrum-table.css

```diff
@@ -12,658 +12,561 @@ governing permissions and limitations under the License.
+            var(--spectrum-table-selected-row-background-color)
+        )
+    );
+    --spectrum-table-selected-cell-background-color-focus: var(
+        --highcontrast-table-selected-row-background-color-focus,
+        var(
+            --mod-table-selected-row-background-color-focus,
+            var(--spectrum-table-selected-row-background-color-focus)
+        )
     );
-    --spectrum-table-spacious-cell-vertical-alignment: var(
-        --spectrum-table-m-spacious-cell-vertical-alignment,
-        top
+}
+:host([quiet]) {
+    --mod-table-border-radius: var(--mod-table-border-radius--quiet, 0px);
+    --mod-table-outer-border-inline-width: var(
+        --mod-table-outer-border-inline-width--quiet,
+        0px
     );
-    --spectrum-table-spacious-border-radius: var(
-        --spectrum-table-m-spacious-border-radius,
-        var(--spectrum-alias-border-radius-regular)
+    --mod-table-header-background-color: var(
+        --mod-table-header-background-color--quiet,
+        var(--spectrum-transparent-white-100)
     );
-    --spectrum-table-spacious-border-size: var(
-        --spectrum-table-m-spacious-border-size,
-        var(--spectrum-alias-border-size-thin)
+    --mod-table-row-background-color: var(
+        --mod-table-row-background-color--quiet,
+        var(--spectrum-transparent-white-100)
     );
 }
+@media (forced-colors: active) {
+    :host {
+        --highcontrast-table-row-background-color: Canvas;
+        --highcontrast-table-row-text-color: CanvasText;
+        --highcontrast-table-divider-color: CanvasText;
+        --highcontrast-table-border-color: CanvasText;
+        --highcontrast-table-icon-color: CanvasText;
+        --highcontrast-table-icon-color-focus: Highlight;
+        --highcontrast-table-selected-row-background-color: Highlight;
+        --highcontrast-table-selected-row-text-color: HighlightText;
+        --highcontrast-table-selected-row-text-color-default: HighlightText;
+        --highcontrast-table-selected-row-background-color-focus: Highlight;
+        --highcontrast-table-selected-row-text-color-focus: HighlightText;
+        --highcontrast-table-row-background-color-hover: Highlight;
+        --highcontrast-table-row-text-color-hover: HighlightText;
+        --highcontrast-table-section-header-text-color: Canvas;
+        --highcontrast-table-section-header-background-color: CanvasText;
+        --highcontrast-table-focus-indicator-color: Highlight;
+        --highcontrast-table-transition-duration: 0;
+    }
+    @supports (color: SelectedItem) {
+        :host {
+            --highcontrast-table-selected-row-background-color: SelectedItem;
+            --highcontrast-table-selected-row-text-color: SelectedItemText;
+            --highcontrast-table-selected-row-text-color-default: SelectedItemText;
+        }
+    }
+}
+:host:not(.spectrum-Table-scroller) {
+    border-collapse: separate;
+    border-spacing: 0;
+}
+:host:not(.spectrum-Table-scroller) {
+    display: table;
+}
```

Comment:
What do we need to do or document about these?

Reply:
Nothing at the moment. Unless you think it's necessary to change the spectrum config to make these just `:host` for now. The `:not` is connected to the [scroller](https://pr-1842--spectrum-css.netlify.app/table#scrollablewithfixedcolumnheaders) that is not yet implemented on the SWC side (which would remove the need to use flex display properties and has an additional wrapper). I don't know if and how you want to implement that feature yet; this PR brings things to parity with the existing flex display w/ scroller.